labs/334_renew_certificates.md

```diff
@@ -82,6 +82,9 @@ You can use the following playbooks to replace the certificates of the other mai
      -    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/redeploy-node-certificates.yml
 -    registry
      -    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/redeploy-registry-certificates.yml
+
+[Warning: Do not replace the router certificates with the following playbook. It will break your routers running on Openshift 3.6. If you like to replace the router certificates, you can use try after upgrading to Openshift 3.7.]
```

Comment:
Remove square brackets. Please consider the following suggestions for phrasing:
"... with the following playbook **as** it will break..."
"Replace the router certificates after upgrading to OpenShift 3.7."